### PR TITLE
Setup communication front end and business rules

### PR DESCRIPTION
**Pull request by rabi-evoco**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

**Review by vialde — APPROVED** [review]

File: charts/pip-frontend/values.dev.template.yaml [hunk 1]
```diff
@@ -2,3 +2,7 @@ nodejs:
   # Don't modify below here
   image: ${IMAGE_NAME}
   ingressHost: ${SERVICE_FQDN}
+  java:
```
Comment (chris-pateman) [inline charts/pip-frontend/values.dev.template.yaml:5]: You do not need the Java node

File: charts/pip-frontend/values.yaml [hunk 2]
```diff
@@ -7,6 +7,7 @@ nodejs:
 #    Publishing-information-project:
 #      secrets:
 #        - AppInsightsInstrumentationKey
+java:
```
Comment (chris-pateman) [inline charts/pip-frontend/values.yaml:10]: You should not have the Java node here

File: charts/pip-frontend/values.yaml [hunk 3]
```diff
@@ -7,6 +7,7 @@ nodejs:
...
   environment:
-#    API_URL: pip-business-rules.{{ .Values.global.environment }}.platform.hmcts.net
-    API_URL: http://localhost:8080
+   API_URL: pip-business-rules.{{ .Values.global.environment }}.platform.hmcts.net > sds-app.{{ .Values.global.environment }}.platform.net
```
Comment (chris-pateman) [inline charts/pip-frontend/values.yaml:12]: You should not have everything after the '>' as it should just be a URL

**Review by chris-pateman — CHANGES_REQUESTED** [review]
need to make sure it is working and deployed

**Review by chris-pateman — APPROVED** [review]

**Pull request merged**